### PR TITLE
fix table layout and colors

### PR DESCRIPTION
**Pull request by adiabuk**
- change row color depending on alert urgency
- Make text visible on all color backgrounds
- Fix hover row color change
- fix line wrap for long lines
- fix column size
- move left-align from job to css
- some cleanup of old code comments
- cleanup of css formatting

### CODE REVIEW

File: jobs/sensu.rb [hunk 1]
```diff
@@ -76,14 +91,18 @@ def get_assoc(column_name, entry)
...
       # add column to event var
-      column_var = { class: 'left', value: data }
+      column_var = { class: status_string, value: data }
       event_var[:cols].insert(-1, column_var)
+
+      #color={class: 'critical'}
+      #event_var[:cols].insert(0, color)
     end
     # add complete row
+   # print event_var
```
Comment (carinadigital) [inline jobs/sensu.rb]: Debug code?

File: widgets/sensu_table/sensu_table.html [hunk 2]
```diff
@@ -6,7 +6,7 @@ <h1 class="title" data-bind="title">
     <option value="platops">Platops</option>
     <option value="auth">Auth</option>
 </select>
-
+<br><br>
```
Comment (carinadigital) [inline widgets/sensu_table/sensu_table.html]: Proabably a cleaner way to leave space than line breaks, e.g. margins

File: widgets/sensu_table/sensu_table.scss [hunk 3]
```diff
@@ -45,37 +46,54 @@ $tr-hover-background-color: #858585;
...
-  tr:nth-child(odd) { background-color: $tr-odd-background-color; }
-  tr:nth-child(even) { background-color: $tr-even-background-color; }
-
-  // Highlight table rows on mouse over
-  tr:hover { background-color: $tr-hover-background-color; }
+  tr:nth-child(odd) { background-color: $tr-odd-background-color; height: 50px; }
+  tr:nth-child(even) { background-color: $tr-even-background-color; height: 50px; }
 
-  td {
+    td {
```
Comment (carinadigital) [inline widgets/sensu_table/sensu_table.scss]: alignment ?

File: jobs/sensu.rb [hunk 4]
```diff
@@ -11,6 +11,19 @@
...
+	when 0
+		return 'ok'
+	when 1
+		return 'warning'
+	when 2
+		return 'critical'
+	else
+		return 'unknown'
+	end
+end
```
Comment (c10l) [inline jobs/sensu.rb]: Are these hard tabs? :scream_cat:

File: widgets/sensu_table/sensu_table.scss (hunk 3 above)
Comment (adiabuk) [inline widgets/sensu_table/sensu_table.scss]: well, spotted, have now reformatted the css due to many misalignments.

File: widgets/sensu_table/sensu_table.html (hunk 2 above)
Comment (adiabuk) [inline widgets/sensu_table/sensu_table.html]: changed to bottom margin for select element just above

File: jobs/sensu.rb (hunk 4 above)
Comment (adiabuk) [inline jobs/sensu.rb]: It looked better in the editor :)  But have changed to spaces.

**Pull request merged**